common/syntaxflow/sfvm/vm.go

```diff
@@ -92,18 +96,27 @@ func (s *SyntaxFlowVirtualMachine) Compile(text string) (frame *SFFrame, ret err
 	if len(errLis.GetErrors()) > 0 {
 		return nil, utils.Errorf("SyntaxFlow compile error: %v", errLis.GetErrorString())
 	}
-
 	result.text = text
 	result.VisitFlow(flow)
 	frame = result.CreateFrame(s.vars)
 	frame.config = s.config
-	frame.debug = s.config.debug
-	frame.Title = result.title
-	frame.Description = result.description
-	frame.AllowIncluded = result.allowIncluded
-	frame.Purpose = result.purpose
-	frame.Language = result.language
-	frame.Severity = result.severity
+	frame.rule.Content = text
+	frame.rule.Title = result.title
+	frame.rule.TitleZh = result.title_zh
+	frame.rule.Description = result.description
+	frame.rule.Type = schema.ValidRuleType(result.purpose)
+	frame.rule.AllowIncluded = func() bool {
```

Comment:
simplify this code. jist `rule.AllowInclude = result.allowInclude!=""` and `if rule.AllowInclude{}`